Terminal.Gui/ConsoleDrivers/ConsoleKeyMapping.cs

```diff
@@ -31,12 +31,12 @@ public static class ConsoleKeyMapping
     ///     for the input language and the high word contains a device handle to the physical layout of the keyboard.
     /// </returns>
     [DllImport ("user32.dll", EntryPoint = "GetKeyboardLayout", CharSet = CharSet.Unicode)]
-    private extern static nint GetKeyboardLayout (nint idThread);
+    private static extern nint GetKeyboardLayout (nint idThread);
 
     //[DllImport ("user32.dll", EntryPoint = "GetKeyboardLayoutNameW", CharSet = CharSet.Unicode)]
     //extern static uint GetKeyboardLayoutName (uint idThread);
-    [DllImport ("user32.dll")] private extern static nint GetForegroundWindow ();
-    [DllImport ("user32.dll")] private extern static nint GetWindowThreadProcessId (nint hWnd, nint ProcessId);
+    [DllImport ("user32.dll")] private static extern nint GetForegroundWindow ();
```

Comment:
I had considered making a rule in ReSharper targeting DllImport and LibraryImport library names to normalize them on extensionless names, as is the recommendation. The runtime does platform-appropriate name mangling on library names. It's less relevant for Windows, but quite relevant for linux, since different distros use different naming conventions for the same libraries, and just making the two follow the same scheme is nice for consistency anyway.

This sort of rule would probably be pretty easy to create and I'm fairly certain ReSharper can be told to apply custom rules when formatting code.